lib/aws/bootstrap/eks-master-cluster.j2.tf

```diff
@@ -6,13 +6,18 @@ locals {
 
 locals {
   tags_eks = {
-    ClusterId = var.kubernetes_cluster_id,
+    ClusterId = var.kubernetes_cluster_id
     ClusterName = var.kubernetes_cluster_name,
     Region = var.region
+    creationDate = time_static.on_cluster_create.rfc3339
```

Comment:
Are you sure it works? Doesn't it change the date on re-apply?

Reply:
Tried it, doesn't change things after first apply.